client/ionic/src/surveys/triage.json

```diff
@@ -13,37 +20,308 @@
+          "isRequired": true,
           "choices": [
             {
-              "value": "fever",
+              "value": "item1",
               "text": "Fever"
             },
             {
-              "value": "cough",
+              "value": "item2",
               "text": "Cough"
             },
             {
-              "value": "breath",
-              "text": "Shortness of breath"
+              "value": "item3",
+              "text": "Difficulty Breathing"
+            },
+            {
+              "value": "item5",
+              "text": "Other"
             }
           ]
         },
+        {
+          "type": "checkbox",
+          "name": "question2",
+          "visibleIf": "{question1} contains 'item5'",
+          "title": "Are you experiencing any of the following symptoms?",
+          "choices": [
+            {
+              "value": "item1",
+              "text": "Muscle Pain"
+            },
+            {
+              "value": "item2",
+              "text": "Nasal Congestion"
+            },
+            {
+              "value": "item3",
+              "text": "Sore Throat"
+            },
+            {
+              "value": "item4",
+              "text": "Headache"
+            }
+          ],
+          "hasNone": true,
+          "noneText": "None of the above"
+        },
         {
           "type": "text",
-          "name": "days",
-          "visibleIf": "{symptoms} notempty",
-          "title": "Approximately how long ago did you first notice symptoms?",
-          "description": "Please enter the number of days",
-          "inputType": "number"
+          "name": "question3",
+          "visibleIf": "{question1} anyof ['item1', 'item2', 'item3'] or {question2} anyof ['item1', 'item2', 'item3', 'item4']",
+          "title": "When did these symptoms start",
+          "inputType": "date"
+        },
+        {
+          "type": "checkbox",
+          "name": "question7",
+          "title": "Do you have the following pre-existing medical conditions",
+          "choices": [
+            {
+              "value": "item1",
+              "text": "Heart Disease"
+            },
+            {
+              "value": "item2",
+              "text": "Diabetes"
+            },
+            {
+              "value": "item3",
+              "text": "Lung Disease"
+            }
+          ]
         },
         {
           "type": "text",
-          "name": "question1",
-          "visibleIf": "{days} > 14"
+          "name": "question5",
+          "visibleIf": "{question1} notempty",
+          "title": "Your Age",
+          "validators": [
+            {
+              "type": "numeric",
+              "text": "must be a number"
+            }
+          ],
+          "inputType": "number"
+        },
+        {
+          "type": "dropdown",
+          "name": "question6",
+          "title": "Your Country",
+          "choices": [
```

Comment:
Shouldn't we extract this from the `countries.json` file? (Or if not, remove that file)

Reply:
I wanted to, but then that list isn't the WHO list. So now I will wait until the localisation PR to  deal with it. Removing it for now